openms-wiki/building-openms.md

```diff
@@ -0,0 +1,11 @@
+# Build OpenMS
```

Comment:
markdown formatting for title:

Build OpenMS from Source
\===================

Reply:
I think "from" should be capitalised